### DISCUSSION TITLE
Can't switch back form Local Mode to Cloud Mode

### DISCUSSION THREAD

**Discussion by AchimTheobald** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#40] [Q&A]
Hello everyone,

Today I put my newly delivered EZ1-M into operation. That worked so far. Unfortunately, I forgot to check and update the firmware before activating "Local Mode".
If I now query the EZ1 data using IP: "get DeviceInfo" everything is ok, only the "devVer" is reported as empty. All power values displayed in HomeAssistant are correct.
A Stone Age firmware (1.0.6) is displayed in the APP.
Unfortunately, I can't find a point in the app to switch back to "Cloud Mode".
So I can no longer establish a connection with the EZ1 via the APP, neither via WiFi nor via BT. Do you have any idea what I can do there?
```json
{
   "data": {
     "deviceId": "E070000XXXXX",
     "devVer": "",
     "ssid": "XXXXXXXX-2.4GHz",
     "ipAddr": "192.168.1.150",
     "minPower": "30",
     "maxPower": "800"
   },
   "message": "SUCCESS",
   "deviceId": "E070000XXXXX"
}
```

**Comment by AchimTheobald** [comment]
Solved...

**Reply by mawoka-myblock** [reply]
Could you please tell us how?

**Reply by AchimTheobald** [reply]
In addition to the smartphone, I installed the APP on the tablet. Now I was able to access it again via “Direct Connection” and switch to “Cloud Mode”.
Unfortunately, I see that it is reported as firmware 1.0.0 and attempting a "firmware upgrade" results in an endless loop with a "flashing hourglass" in the middle.
Tonight I'll try out the described procedure "Firmware update instructions" under "Issues" and tomorrow I'll attach two dummy power supplies to the EZ1 instead of the PV panels and hope for the best.

![20240504_164930](https://github.com/SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant/assets/126383714/7714c9b1-cd16-4955-b9d5-36382baafd0d)

**Reply by mawoka-myblock** [reply]
One power supply is enough to keep the inverter active.
